src/components/icon/assets/chatLeft.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import * as React from 'react';
+const OuiIconChatLeft = ({ title, titleId, ...props }) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width={16}
+    height={16}
```

Comment:
Is it a norm among to have the height and width hard-coded, despite having a `viewBox`? If it is a norm, there are some assets that are missing it.

Reply:
your guess is as good as mine - I searched and saw that many SVG icon files had them, but just about as many didn't. Note that these `*.js` files are auto-compiled from the SVGs, so it's based on whether those props are set there.

Reply:
I can remove them if you think it would be better - I was mostly using https://github.com/opensearch-project/oui/pull/658 as my guide.

Reply:
Having them would make controlling the dimensions challenging. If we don't have a reason to have these hardcoded, let's remove them at least for these new assets (and decide on the others later).

Reply:
Thanks @AMoo-Miki I removed them in the latest commit.